test/e2e/testnet/node.go

```diff
@@ -148,6 +151,9 @@ func NewNode(
 		return nil, err
 	}
 	args := []string{"start", fmt.Sprintf("--home=%s", remoteRootDir), "--rpc.laddr=tcp://0.0.0.0:26657"}
+	if !enableBBR && equalOrHigher(version, appVersiopnWithBBRFlag) {
+		args = append(args, "--force-no-bbr")
```

Comment:
> definitely not blocking, but imo we can just always add this flag since we are not doing network tests on knuu.

This was what I initially thought as well, but validators were failing in `MinorVersionCompatibility` test complaining that `--force-no-bbr` flag is not available/defined. That's why I added this check to see if the version supports that flag or not.

Reply:
An update on my previous comment: we currently have no releases with the BBR flag enabled. Therefore, the only test that requires this flag is `E2ESimple`; the rest should not use it, as it is undefined. The benchmark tests, however, will handle this differently. Since they are meant to always reflect the current state of the celestia-app, the flag should always be passed in those cases (unless an older version is used, which is not currently the case).